File: fix-conflicts-simple.js
import fs from 'fs';
import { execSync } from 'child_process';

function fixAllMergeConflicts() {
  try {
    // Find all files with merge conflicts
    const result = execSync('grep -r "^<<<<<<<\\|^=======\\|^>>>>>>>" . --include="*.tsx" --include="*.ts" --include="*.js" --include="*.jsx" -l', { encoding: 'utf8' });
    const files = result.trim().split('\n').filter(f => f);
    
    console.log(`Found merge conflicts in ${files.length} files:`, files);
    
    files.forEach(filePath => {
      try {
        let content = fs.readFileSync(filePath, 'utf8');
        
        // Simple approach: remove all merge conflict markers and keep only the content after =======
        const lines = content.split('\n');
        const resolvedLines = [];
        let skipUntilNextMarker = false;
        
        for (let i = 0; i < lines.length; i++) {
          const line = lines[i];
          
          if (line.trim().startsWith('<<<<<<<')) {
            skipUntilNextMarker = true;
            continue;
          } else if (line.trim().startsWith('=======')) {
            skipUntilNextMarker = false;
            continue;
          } else if (line.trim().startsWith('>>>>>>>')) {
            skipUntilNextMarker = false;
            continue;
          }
          
          if (!skipUntilNextMarker) {
            resolvedLines.push(line);
          }
        }
        
        const resolvedContent = resolvedLines.join('\n');
        fs.writeFileSync(filePath, resolvedContent);
        console.log(`Fixed merge conflicts in ${filePath}`);
        
      } catch (error) {
        console.error(`Error processing ${filePath}:`, error.message);
      }
    });
    
  } catch (error) {
    console.error('Error finding files with merge conflicts:', error.message);
  }
}

fixAllMergeConflicts();

File: fix_conflicts_simple.py
#!/usr/bin/env python3
import os
import re
import glob

def fix_conflicts(file_path):
    try:
        with open(file_path, 'r', encoding='utf-8') as f:
            content = f.read()
        
        # Remove all merge conflict markers and choose the newer version (after 
        
        # Also handle cases without (just remove the markers)
        new_content = re.sub(r'        
        if new_content != content:
            with open(file_path, 'w', encoding='utf-8') as f:
                f.write(new_content)
            print(f"Fixed conflicts in {file_path}")
            return True
        return False
    except Exception as e:
        print(f"Error processing {file_path}: {e}")
        return False

def main():
    files = glob.glob('**/*.ts', recursive=True) + glob.glob('**/*.tsx', recursive=True)
    files = [f for f in files if not any(exclude in f for exclude in ['node_modules', '.git'])]
    
    fixed_count = 0
    for file_path in files:
        if fix_conflicts(file_path):
            fixed_count += 1
    
    print(f"Fixed conflicts in {fixed_count} files")

if __name__ == "__main__":
    main()